### PR TITLE
executor: remove Next function for HashAggExec

### PR DESCRIPTION
**Pull request by lwhile**

### CODE REVIEW

File: executor/aggregate.go [hunk 1]
```diff
@@ -143,37 +143,7 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 
 // Next implements the Executor Next interface.
 func (e *HashAggExec) Next(ctx context.Context) (Row, error) {
-	// In this stage we consider all data from src as a single group.
-	if !e.executed {
-		for {
-			hasMore, err := e.innerNext(ctx)
```
Comment (zz-jason) [inline executor/aggregate.go]: can we remove `innerNext` ?

Comment (lwhile) [reply executor/aggregate.go]: I think so, let me remove it.

**Comment by zz-jason** [comment]
@lwhile please fix ci.

**Comment by zz-jason** [comment]
to #5985

**Comment by lwhile** [comment]
@zz-jason ok, I will fix it tonight or tomorrow

**Comment by lwhile** [comment]
@zz-jason Can I remove all the ```Next``` test in ```executor/executor_test.go```?

**Comment by zz-jason** [comment]
@lwhile No. We should rewrite them instead.

**Comment by lwhile** [comment]
@zz-jason  I am confused about the output of CI and I have no debug environment.Please give me some Infomation for fix CI?

**Comment by XuHuaiyu** [comment]
@lwhile You can use `make dev` to check whether your code could pass the UTs.

**Comment by XuHuaiyu** [comment]
FAIL: admin_test.go:204: testSuite.TestScan

admin_test.go:287:
    s.testIndex(c, ctx, db.L, tb, tb.Indices()[0])
admin_test.go:356:
    c.Assert(err, IsNil)
... value *errors.Err = &errors.Err{message:"", cause:(*errors.Err)(0xc420fdb040), previous:(*errors.Err)(0xc420fdb040), file:"/go/src/github.com/pingcap/tidb/util/admin/admin.go", line:199} ("can not get count, sql SELECT COUNT(*) FROM `test_admin`.`t` USE INDEX(`c`) result rows 0")

**Comment by zz-jason** [comment]
@lwhile any update ?

**Comment by shenli** [comment]
@lwhile A friendly ping. Please fix the CI.

**Comment by lwhile** [comment]
@shenli Sorry, I will try to fix the errors in the next few days :)

**Comment by lwhile** [comment]
@shenli Hi, I could pass all the test in my local environment when I merge the latest branch master to my patch, but the online CI says 
```
./executor/aggregate.go
Makefile:75: recipe for target 'check' failed
make: *** [check] Error 1
Exited with code 2
```
Could give me more tips?

**Comment by XuHuaiyu** [comment]
You can run `go fmt ./...` in `executor` directory.

**Comment by tiancaiamao** [comment]
What's your Go version? You need the latest Go to format the source files. @lwhile

**Comment by lwhile** [comment]
Can pass the CI check now. @shenli 

Thanks for your help. I am a reader of your blog, and your article is very well. @tiancaiamao

**Comment by XuHuaiyu** [comment]
LGTM
PTAL @zz-jason

File: executor/aggregate.go [hunk 2]
```diff
@@ -143,37 +143,7 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 
 // Next implements the Executor Next interface.
 func (e *HashAggExec) Next(ctx context.Context) (Row, error) {
```
Comment (zz-jason) [inline executor/aggregate.go]: we can more `Next` function totally now.

Comment (lwhile) [reply executor/aggregate.go]: removed

**Comment by lwhile** [comment]
@zz-jason The Jenkins test didn't pass.Should I fix it?

**Comment by zz-jason** [comment]
@lwhile It's a known issue, not your fault, we are working on it

**Pull request merged**